src/AppBundle/Service/EventProcessor.php

```diff
@@ -352,6 +359,20 @@ private function setItemsCreatedOrImprovedOnWikidata(EventWiki $wiki, EventWikiR
         $this->createOrUpdateEventStat('items-improved', $ret['edited']);
     }
 
+    private function setUserCounts(EventWiki $wiki, EventWikiRepository $ewRepo): void
```

Comment:
Not sure why phpcs didn't complain, but this method needs a docblock :)

Reply:
Ah! Are we expected to document private methods too? It feels right to me to document everything, but I get that if this were a library, private methods need not be explained because they shouldn't/can't be used.

Reply:
Done.